test_conformance/images/clFillImage/test_loops.cpp

```diff
@@ -62,6 +67,12 @@ int test_image_type( cl_device_id device, cl_context context, cl_command_queue q
             break;
         default: log_error("Unhandled method\n"); return -1;
     }
+    else if (testMethod == k1DBuffer)
```

Comment:
This is failing compilation since there's no 'if' to match the 'else if'
Also, from reviewing the code it looks like k1DBuffer would hit the default and error out at line 68?

Reply:
I'm seeing the same build errors.

Reply:
Sorry about that, it was a rebase error on my side.